sdv/constraints/base.py

```diff
@@ -285,7 +285,7 @@ def filter_valid(self, table_data):
             LOGGER.debug('%s: %s invalid rows out of %s.',
                          self.__class__.__name__, sum(~valid), len(valid))
 
-        return table_data[valid]
+        return table_data[valid.values]
```

Comment:
@fealho would it be possible to add a unit or integration test that reproduces the error?
I would add the link of the issue in the test docstring and also make sure that it fails before making this change and it is fixed afterwards.